neurow/lib/neurow/public_api/endpoint.ex

```diff
@@ -244,10 +247,26 @@ defmodule Neurow.PublicApi.Endpoint do
 
           # Consume useless messages to avoid memory overflow
           _ ->
-            conn |> loop(sse_timeout_ms, keep_alive_ms, last_message_ms, last_ping_ms, jwt_exp_s)
+            conn
```

Comment:
**Suggestion** 🤔 : i we do one pipe, i think it's clearer to have just put the param directly in the func call